src/sos/__main__.py

```diff
@@ -1546,15 +1538,15 @@ def cmd_execute(args, workflow_args):
         if any(x in ("failed", "aborted") for x in res):
             for t, s in zip(args.tasks, res):
                 if s in ("failed", "aborted") and t not in failed_tasks:
-                    env.logger.warning("{} ``{}``".format(t, s))
+                    env.logger.warning(f"{t} ``{s}``")
                     failed_tasks.add(t)
             if all(x in ("completed", "failed", "aborted") for x in res):
-                raise RuntimeError(
-                    "{} completed, {} failed, {} aborted)".format(
-                        len([x for x in res if x == "completed"]),
-                        len([x for x in res if x == "failed"]),
-                        len([x for x in res if x.startswith("aborted")]),
-                    ))
+                into_runtime_error = (
```

Comment:
what is this?

Reply:
It is a variable that stores all of the f strings so that they do not have to be contained in a single line of code within the RuntimeError.

Reply:
where are the "completed", "failed", "aborted" etc?